src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -135,6 +122,41 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
     return this._disabled;
   }
 
+  /**
+   * Indicated whether to hide the flag in the input element.
+   * @default "false"
+   */
+  @Input()
+  public hideSelectedCountryFlag: boolean;
+
+  /**
+   * Indicated whether to include phone information in the selected country and country dropdown.
+   * @default "false"
+   */
+  @Input()
```

Comment:
Do we need a description for this property?

Reply:
Done. That was an existing import that just got moved around but it definitely should have had JSDocs